app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
```

Comment:
The use of `split("/")` is not cross-platform. Consider using `os.path.join` to concatenate parts of the path as recommended in the checklist to ensure compatibility across different operating systems【4:2†checklist.md】.